packages/data-connect/package.json

```diff
@@ -31,10 +31,11 @@
     "prettier": "prettier --write '*.js' '*.ts' '@(src|test)/**/*.ts'",
     "build:deps": "lerna run --scope @firebase/'{app,data-connect}' --include-dependencies build",
     "dev": "rollup -c -w",
-    "test": "run-p --npm-path npm test:emulator",
+    "test": "run-p --npm-path npm lint test:emulator",
     "test:ci": "node ../../scripts/run_tests_in_ci.js -s test:emulator",
-    "test:all": "run-p --npm-path npm lint test:unit",
-    "test:browser": "karma start --single-run",
+    "test:browser:unit": "karma start --unit",
```

Comment:
I don't see a "unit" command line option in packages/data-connect/karma.conf.js like auth has https://github.com/firebase/firebase-js-sdk/blob/main/packages/auth/karma.conf.js#L37 so I don't think this does anything? Also do you want to remove --single-run? Doesn't that make the browser not exit?